### ISSUE TITLE
Resize applets in systray?

### ISSUE THREAD

**Issue by bam80** [Zren/zren.github.io#9]
I wasn't able to resize my applet being loaded to systray:  `Layout` type seem influences it's appearance in desktop/panels only.
I wonder why is that, and what is the proper way to resize individual applets in systray, if any.
Maybe the explanation could find it's place in you Plasma Widget Tutorial.
Thanks.

**Comment by Zren** [comment]
The system tray is a containment that displays the child widgets in a fixed size. The child widget cannot tell the system tray what size to display the popup dialog.

But ya, I should writeup a section about the system tray (there's basically one or two metadata entries necessary to turn it into one).

**Comment by bam80** [comment]
Thank you for a quick answer!
Just for clarity: I tried to change size of applet's CompactRepresentation, to change it's icon not popup.
From your answer I can conclude it's impossible.

**Comment by Zren** [comment]
Correct. The icons are also a fixed size set by the system tray. It uses a `GridView` which forces a fixed size in all grid cell delegates.

* https://invent.kde.org/plasma/plasma-workspace/-/blob/master/applets/systemtray/package/contents/ui/main.qml#L158-175
* https://doc.qt.io/qt-5/qml-qtquick-gridview.html
